test/parse-args.js

```diff
@@ -44,14 +64,16 @@ describe('parse-args', () => {
       argv.lines.should.be.equal(95)
     })
     it('should use config file specified in --config', () => {
+      const tmpDir = resolve(process.cwd(), 'foo')
       const argv = buildYargs().parse(['node', 'c8', '--config', require.resolve('./fixtures/config/.c8rc.json')])
       argv.lines.should.be.equal(101)
-      argv.tempDirectory.should.be.equal('./foo')
+      argv.tempDirectory.should.be.equal(tmpDir)
```

Comment:
I don't like that we have a behavioural change as a result of these updates, coverage should ideally just be changing coverage.

I don't remember if there's a good reason why I don't `resolve` the path of `tmpDir`, but if there are no outstanding bugs resulting from the relative path, I don't think we should change it (especially in a PR that is simply attempting to increase coverage).